core/lib/spree/core/controller_helpers/auth.rb

```diff
@@ -42,10 +42,10 @@ def redirect_back_or_default(default)
 
         def set_guest_token
           unless cookies.signed[:guest_token].present?
-            cookies.permanent.signed[:guest_token] = {
+            cookies.permanent.signed[:guest_token] = Spree::Config[:guest_token_cookie_options].merge(
```

Comment:
If we want users to be able to also change `value` and `httponly` options we could merge the config hash into the default one but I'm not sure if that could be actually something needed. What do you think?

Reply:
We must never allow this. See comment below 👇